T_PHY_002 modified to address explicitely fault injection attacks (#50)
riscv-non-isa/riscv-security-model: specification/src/chapter2.adoc

a| * Implement robust power management and radiation control
* Data Independent Execution Latency (Zkt, Zvkt)

| T_PHY_002
| Physical memory manipulation
| Direct +
Logical or physical
a| * Using NVDIMM, interposers, or physical probing to read, record, or replay
physical memory.
a| * Implement robust memory error detection, cryptographic memory protection,
or physical tamper resistance.

> * Supervisor domain ID, privilege level, or MTT attributes, may be used to

Reviewer: ```suggestion
* Supervisor domain ID, privilege level, or MPT attributes, may be used to
```

* Data Independent Execution Latency (Zkt, Zvkt)

| T_PHY_002

> | Fault Injection

Reviewer 2: I would prefer we keep both: Physical Memory Manipulation and Fault Injection. Both are meaningful

Reviewer: agree - add a T_PHY_003

Reviewer 3: IMO FI is a separate threat from most of the physical memory manipulation as described (probing, interposers etc) whereas FI is modifying circuit state  (EMFI, laser, voltage/clock/reset glitch etc ). Row hammer is FI specific to memory, and isn't really physical manipulation, so perhaps we  add a T_PHY_03 - fault Injection attack, and move row hammer under that ?

| Direct +
Logical or Physical
a| * Row-hammer type software attacks to manipulate nearby memory cells  
* Using NVDIMM, interposers, or physical probing to read, record, or replay physical memory  
* Fault injection attacks (glitching, laser, electromagnetic, ...) to extract hardware-provisioned assets  
a| * Implement robust memory error detection, cryptographic memory protection, or physical tamper resistance  

> * Use supervisor domain ID, privilege level, or MTT attributes to derive memory encryption contexts at the domain or workload granularity  

Reviewer: ```suggestion
* Use supervisor domain ID, privilege level, and/or MPT permissions to derive memory access control / encryption contexts at the domain or workload granularity  
```

Reviewer 3: FI isn't only used to extract hardware provisioned state but also to modify circuit behaviour, bypassing isolation or confidentiality protections

| Deploy appropriate governance, accreditation, and certification processes for
an ecosystem.

| T_PHY_005
| Fault Injection
| Direct or remote +
Logical or physical
a| * Rowhammer-type software attacks to manipulate nearby memory cells.
* Fault injection attacks (glitching, laser, electromagnetic, etc.) to extract hardware-provisioned assets, modify the control flow, circumvent countermeasures, etc.
a| * Implement robust memory error detection, cryptographic memory protection, or physical tamper resistance. 
* Provide a level of tamper resistance, e.g., through RoT attestation, redundancy during execution, etc.
* Enforce proper access control for the DVFS configuration.
* Employ lockstep processors for security-critical devices.

> 

Reviewer 3: suggest the use of sensors ?

Reviewer 3: ```suggestion
* Employ physical sensors to detect attack
```

Reviewer 3: IGTM
